app/src/main/java/ch/epfl/qedit/model/User.java

```diff
@@ -86,17 +94,43 @@ public String getFullName() {
         return firstName + " " + lastName;
     }
 
-    public Role getRole() {
-        return role;
+    public int getScore() {
+        return score;
+    }
+
+    public int getSuccess() {
+        return success;
+    }
+
+    public int getAttempt() {
+        return attempt;
+    }
+
+    public void incrementScore(int points) {
+        if (points < 0)
+            throw new IllegalArgumentException("Cannot increment score with negative points");
+        score += points;
```

Comment:
Maybe when we increment score we should increment success and attempt in the same time

Reply:
to be fair, this i am not sure yet, i prefer to let the personne who use the user class to decide how he want to handle the increment of score. However i can increment attempte when i increment success

Reply:
No problem ;).